new/rfc-meetbot.md

```diff
@@ -0,0 +1,65 @@
+---
+RFC: unassigned
+Author: JJ Asghar <jj@chef.io>
+Status: Draft
+Type: Process
+---
+
+# Enabling meetbot for Chef IRC Community meetings.
+
+I would like to enable [meetbot][meetbot] for the Chef IRC Community Meetings.
+The ability to automatically take notes and executive summaries can open up
+people to have conversations instead of the dogma of making sure the conversations
+stay on track.
+
+## Motivation
+
+    As a community member of Chef,
+    I want to have a higher level of engagement from everyone,
+    so that we automate the human process away, and the bot does the note taking.
```

Comment:
nit, " and the bot does the note taking" doesn't feel like it belongs in the user story.

As a community member of Chef,
I want the meeting notes to be automated,
so that the secretary can participate fully in the discussion.